tests/acceptance/alias_store_test.go

```diff
@@ -0,0 +1,170 @@
+// +build acceptance
+
+package acceptancetests
+
+import (
+	"context"
+	"math/rand"
+	"os"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/consensys/quorum-key-manager/pkg/common"
+	aliasmodels "github.com/consensys/quorum-key-manager/src/aliases/models"
+	aliasstore "github.com/consensys/quorum-key-manager/src/aliases/store"
+	aliaspg "github.com/consensys/quorum-key-manager/src/aliases/store/postgres"
+)
+
+func TestAliasStore(t *testing.T) {
+	s := new(aliasStoreTestSuite)
+	ctx, cancel := context.WithCancel(context.Background())
+
+	sig := common.NewSignalListener(func(signal os.Signal) {
+		cancel()
+	})
+	defer sig.Close()
+
+	var err error
+	s.env, err = NewIntegrationEnvironment(ctx)
```

Comment:
You didn't do what we discussed ? Are you planning on doing it in a different PR? IMO it's a really quick change because here you are basically duplicating everything by creating a new `IntegrationEnvironment`.

Reply:
The idea was to do it in another PR.
I actually started putting it in the existing test runner, but it was putting it under a StoreTest, which didn't make sense since alias is not related to any Store (AKV, AWS KMS, Hashicorp Vault).
So it's not really a quick change because we need to create a higher level test runner that would run the Store on one side, and the alias tests on the other side.

Reply:
https://github.com/ConsenSys/quorum-key-manager/issues/249 will fix it